### PR TITLE
As_98 Index Date Field

### PR DESCRIPTION
**Pull request by YuriHechter**

### CODE REVIEW

File: es/mappings/make.py [hunk 1]
```diff
@@ -219,6 +219,10 @@ def IsCyrillic(lang, something):
         "result_type": {
           "type": "keyword",
         },
+        # Document index date.
+        "index_date": {
+          "type": "date",
```
Comment (bbfsdev) [inline es/mappings/make.py]: Why not:
"format": "strict_date",
As in effective_date?

Comment (bbfsdev) [reply es/mappings/make.py]: I think we should use here:
"epoch_millis"

Comment (YuriHechter) [reply es/mappings/make.py]: done

**Comment by bbfsdev** [comment]
Manually merged.

**Pull request closed without merging**